bosh-release/README.md

```diff
@@ -131,28 +47,37 @@ A BOSH release for deploying WSO2 Identity Server 5.3.0 on BOSH Director:
     Instance                                       Process State  AZ  IPs           VM CID                                VM Type
     wso2is/08b2075d-c7e6-49f8-b223-12d989b734c2  running        -   10.244.15.21  84cac420-fd02-4884-5821-0fad60e3ce29  wso2is-resource-pool
     ...
-
-    # WSO2 Identity Server URL: http://10.244.15.21:9763/carbon/
+    ```
+    To ssh to the instance
+    ```
+    bosh -e vbox -d wso2is ssh wso2is/08b2075d-c7e6-49f8-b223-12d989b734c2
+    ```
+    Access the management console with URL
+    ```
+    http://10.244.15.21:9763/carbon/
```

Comment:
Have we set the hostname in the carbon.xml file? If so, we may need to ask the user to use the hostname with an entry in the /etc/hosts file or a DNS entry. When  this is deployed with PCF, PCF router will provide a domain name and that needs to be added in the product.

Reply:
We have not configured the carbon.xml file here.

Reply:
Noted, let's do that in a future task when we add scalability features.